### ISSUE TITLE
Raspberry Pi build broken?

### ISSUE THREAD

**Issue by slouken** [libsdl-org/SDL#8050]
I'm trying to build on Raspberry Pi, and it's not finding headers I would expect.

For example, I'm trying to build with libusb:
```
/usr/bin/cc   -D_GNU_SOURCE=1 -o CMakeFiles/cmTC_3cf20.dir/CheckIncludeFile.c.o -c /home/saml/projects/SDL/build-rpi/CMakeFiles/CMakeScratch/TryCompile-Y2whUE/CheckIncludeFile.c
/home/saml/projects/SDL/build-rpi/CMakeFiles/CMakeScratch/TryCompile-Y2whUE/CheckIncludeFile.c:1:20: fatal error: libusb.h: No such file or directory
 #include <libusb.h>

./CMakeCache.txt:PC_LIBUSB_CFLAGS:INTERNAL=-I/usr/include/libusb-1.0
./CMakeCache.txt:PC_LIBUSB_INCLUDE_DIRS:INTERNAL=/usr/include/libusb-1.0
./CMakeCache.txt:PC_LIBUSB_MODULE_NAME:INTERNAL=libusb-1.0
./CMakeCache.txt:PC_LIBUSB_STATIC_CFLAGS:INTERNAL=-I/usr/include/libusb-1.0
./CMakeCache.txt:PC_LIBUSB_STATIC_INCLUDE_DIRS:INTERNAL=/usr/include/libusb-1.0

$ ls /usr/include/libusb-1.0
libusb.h
```

Also:
```
-- Found RPi_BcmHost: /opt/vc/lib/libbcm_host.so  
-- Could NOT find RPi_BrcmEGL (missing: RPi_BrcmEGL_EGL_eglext_brcm_h_PATH) 
```
and I notice that FindRPi_BrcmEGL.cmake sets RPi_BrcmEGL_EGL_eglplatform_h_PATH, not RPi_BrcmEGL_EGL_eglext_brcm_h_PATH.
Where did we get this file and why are we using it instead of what we had before?

**Comment by madebr** [comment]
The `SDL_HIDAPI_LIBUSB` error was due to a last-minute fix in my cmake 3.16 pr.

About rpi, I added the find modules to avoid executing a test compile for detecting the platform.
Those find modules were created by me, using https://github.com/raspberrypi/firmware/ as reference.

I created #8052 that should fix both issues. Please test.

**Comment by slouken** [comment]
Thanks!
